src/resource.js

```diff
@@ -22,29 +34,35 @@ module.exports = function (settings, method) {
     // create new Observable resource
     const options = getOptions(settings, this._rx, params.rx);
     const source = getSource(this[method].bind(this), arguments);
-    const stream = source.concat(
-      source.exhaustMap(data => {
+    const stream = source.pipe(
+      concatMap(data => {
         // Filter only data with the same id
-        const filter = current => current[options.idField] === data[options.idField];
+        const filterFn = current => current[options.idField] === data[options.idField];
         // `removed` events get special treatment
-        const filteredRemoves = this.removed$.filter(filter);
+        const filteredRemoves$ = this.removed$.pipe(filter(filterFn));
         // `created`, `updated` and `patched`
-        const filteredEvents = Observable.merge(
+        const filteredEvents$ = merge(
           this.created$,
           this.updated$,
           this.patched$
-        ).filter(filter);
+        ).pipe(
+          filter(filterFn)
+        );
 
-        return Observable.merge(
+        const combinedEvents$ = merge(
           // Map to a callback that merges old and new data
-          filteredEvents,
+          filteredEvents$,
           // filtered `removed` events always mapped to `null`
-          filteredRemoves.mapTo(null)
+          filteredRemoves$.pipe(mapTo(null))
+        );
+
+        return of(data).pipe(
+          concat(combinedEvents$)
         );
       }));
 
     // apply `let` function if set
-    const letStream = options.let ? stream.let(options.let) : stream;
+    const letStream = options.let ? stream.pipe(options.let) : stream;
```

Comment:
I think `options.let` should become `options.pipe` for consistency. Also it would encourage using the new style operators and this is major version anyway

Reply:
Done, I updated the PR.